### ISSUE TITLE
Improve Predis warning

### ISSUE THREAD

**Issue by tillkruss** [rhubarbgroup/redis-cache#229]
Make message more helpful and link to Wiki page.

- What is Predis 
- Why is it dead
- Why switch
- How to switch: Credis/PhpRedis

**Comment by beautifulcode** [comment]
Hi there! Agreed on the above. I'm actually trying to switch from predis to credis on Heroku, but the connection params don't seem to be working... Do you know if Credis supports heroku-redis style REDIS_URL and/or the various WP_REDIS_* env vars? I'm digging through your source and theirs but not sure if it's possible as is. Thanks for all your work!

**Comment by tillkruss** [comment]
I know that Heroku supports PhpRedis, use that 👍🏻

**Comment by tillkruss** [comment]
@beautifulcode check this out:

```php
Config::define('...', [
    'host' => parse_url(env('REDIS_URL'), PHP_URL_HOST),
    'port' => parse_url(env('REDIS_URL'), PHP_URL_PORT),
    'password' => parse_url(env('REDIS_URL'), PHP_URL_PASS),
]);
```

**Comment by tillkruss** [comment]
Closed with c883240.
